app/src/lib/stores/pull-request-store.ts

```diff
@@ -14,6 +14,16 @@ import {
   PullRequestStatus,
 } from '../../models/pull-request'
 import { Emitter, Disposable } from 'event-kit'
+import { Repository } from '../../models/repository'
+import { getRemotes, removeRemote } from '../git'
+import { IRemote } from '../../models/remote'
+
+/**
+ * This is the magic remote name prefix
+ * for when we add a remote on behalf of
+ * the user.
+ */
+export const ForkedRemotePrefix = 'github-desktop-'
```

Comment:
:memo: Could we incorporate some public-facing docs into this PR? It'd be nice to summarize what we're adding to a repository in terms of remotes and branch prefixes, so we can easily refer people with questions and issues to it.